sdk/storage/azure-storage-file-share/tests/test_share.py

```diff
@@ -1,42 +1,41 @@
-# coding: utf-8
-
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
 import os
+import requests
```

Comment:
nit: `requets` is 3rd party library and therefore was in the correct spot previously :)

Reply:
Drats. Fixed 😄